vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-testbench/src/main/java/com/vaadin/flow/component/datetimepicker/testbench/DateTimePickerElement.java

```diff
@@ -247,14 +249,26 @@ public boolean isAutoOpen() {
         return !getPropertyBoolean("autoOpenDisabled");
     }
 
-    private TestBenchElement getDatePicker() {
-        return $("vaadin-date-picker").withAttribute("slot", "date-picker")
-                .first();
+    /**
+     * Get the DatePickerElement for date picker part.
+     *
+     * @return DatePickerElement
+     */
+    public DatePickerElement getDatePicker() {
+        return this.$("vaadin-date-time-picker-date-picker")
```

Comment:
If this is for `main` branch then you just need to use regular `vaadin-date-picker` and `vaadin-time-picker`.
Custom extensions were only used until V23 and have been removed in V24.

Reply:
Thanks